### PR TITLE
Adding slashes for tags when needed according to HTML's spec

### PR DESCRIPTION
**Pull request by Chrissdroid**
Updating files from the `/en/` docs regarding self closing tags technically wrong (according to spec) when specified in the docs

- changing `<slot>` to `<slot />`
- ~changing `` to ``~ — reverted
- changing `<Markdown>` to `<Markdown />`
- ~changing `<meta>` to `<meta />`~ — reverted

(contain also change for `migrate.md` removing useless spaces at end of line, (ty my IDE))

### CODE REVIEW

**Comment by sarah11918** [comment]
I don't think the `<meta>` and `` tags should have a closing slash. But the components (slot, Markdown) definitely should!

**Comment by Chrissdroid** [comment]
When I was browsing for information on this, it said that img and meta *has* to have an self closing according to the spec

But it's from the XML part of the HTML spec so both are "suggested" as correct

I can revert these changes if you prefer the other syntax

**Comment by delucis** [comment]
I’ll leave the final call on this to you @sarah11918. I’m OK either way — I agree that this is most important for the Astro-specific components.

**Comment by sarah11918** [comment]
Well, how's this for the basis of a decision?  Our new basics starter uses `<meta>` tags.  :)

So, I'm going to say let's only change the components for now, and we'll work on a proper style gude where we can make these decisions, and make them available to everyone.

**Comment by sarah11918** [comment]
@Chrissdroid For now, we're only going to use trailing slashes in components, not HTML elements. But, we'll pick something for the style guide. So, I'll merge this when you take them out of img and meta, please.  :)

**Comment by Chrissdroid** [comment]
I'll do !

**Review by delucis — APPROVED** [review]
LGTM!

**Pull request merged**
